--- IXSI-ger.xsd.old	2014-10-10 12:56:39.927328809 +0200
+++ IXSI-ger.xsd	2014-10-10 11:02:51.217008514 +0200
@@ -383,6 +383,12 @@
 		</xs:annotation>
 		<xs:restriction base="xs:NMTOKEN"/>
 	</xs:simpleType>
+        <xs:simpleType name="ConsumptionIDType">
+                <xs:annotation>
+                        <xs:documentation>Aufzählung von Verbrauchsdatentypen.</xs:documentation>
+                </xs:annotation>
+                <xs:restriction base="xs:NMTOKEN"/>
+        </xs:simpleType>
 	<xs:simpleType name="AttributeIDType">
 		<xs:annotation>
 			<xs:documentation>Aufzählung von Attributcodes.</xs:documentation>
@@ -854,6 +860,43 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:complexType>
+        <xs:complexType name="ConsumptionType">
+                <xs:annotation>
+                        <xs:documentation>Typ für einen Rechnungsposten der Verbrauchsdaten.</xs:documentation>
+                </xs:annotation>
+                <xs:sequence>
+                        <xs:element name="Type" type="ConsumptionIDType">
+                                <xs:annotation>
+                                        <xs:documentation>Typ des Verbrauchs.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                        <xs:element name="Name" type="xs:string">
+                                <xs:annotation>
+                                        <xs:documentation>Druckbarer Name des Rechnungsposten.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                        <xs:element name="Unit" type="xs:string" minOccurs="0">
+                                <xs:annotation>
+                                        <xs:documentation>Druckbarer Name der Einheit.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                        <xs:element name="Value" type="xs:string">
+                                <xs:annotation>
+                                        <xs:documentation>Verbrauchswert.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                        <xs:element name="Price" type="EuroPriceType" minOccurs="0">
+                                <xs:annotation>
+                                        <xs:documentation>Preis in Euro-Cent.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                        <xs:element name="Description" type="xs:string" minOccurs="0">
+                                <xs:annotation>
+                                        <xs:documentation>Druckbare Beschreibung des Tarifdetails.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                </xs:sequence>
+        </xs:complexType>
 	<xs:complexType name="OriginDestType">
 		<xs:annotation>
 			<xs:documentation>Typ für den Ausgangspunkt oder den Reisezielort.</xs:documentation>
@@ -1483,6 +1526,97 @@
 			</xs:extension>
 		</xs:complexContent>
 	</xs:complexType>
+        <xs:annotation>
+                <xs:documentation>**** Service (7) : Verbrauchsdaten-Abonnement ****</xs:documentation>
+        </xs:annotation>
+        <xs:complexType name="ConsumptionSubscriptionRequestType">
+                <xs:annotation>
+                        <xs:documentation>Anfrage um Verbrauchsdaten für gegebene Buchungen zu abonnieren oder zu kündigen.</xs:documentation>
+                </xs:annotation>
+                <xs:sequence>
+                        <xs:element name="BookingID" type="BookingIDType" maxOccurs="unbounded">
+                                <xs:annotation>
+                                        <xs:documentation>Verbrauchsdaten für Buchungen abonnieren.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                        <xs:element name="Unsubscription" type="xs:boolean" default="false" minOccurs="0">
+                                <xs:annotation>
+                                        <xs:documentation>Falls gesetzt, gegebene Abos der Buchungen kündigen. Andernfalls abonnieren.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                </xs:sequence>
+        </xs:complexType>
+        <xs:complexType name="ConsumptionSubscriptionResponseType">
+                <xs:annotation>
+                        <xs:documentation>Antwort um Verbrauchsdaten für gegebene Buchungen zu abonnieren.</xs:documentation>
+                </xs:annotation>
+                <xs:complexContent>
+                        <xs:extension base="AbstractBaseResponseType"/>
+                </xs:complexContent>
+        </xs:complexType>
+        <xs:complexType name="ConsumptionSubscriptionStatusRequestType">
+                <xs:annotation>
+                        <xs:documentation>Anfrage um alle abonnierten Verbrauchsdaten zu erhalten.</xs:documentation>
+                </xs:annotation>
+        </xs:complexType>
+        <xs:complexType name="ConsumptionSubscriptionStatusResponseType">
+                <xs:annotation>
+                        <xs:documentation>Antwort um alle abonnierten Verbauchsdaten zu erhalten.</xs:documentation>
+                </xs:annotation>
+                <xs:complexContent>
+                        <xs:extension base="AbstractBaseResponseType">
+                                <xs:sequence>
+                                        <xs:element name="BookingID" type="BookingIDType" minOccurs="0" maxOccurs="unbounded">
+                                                <xs:annotation>
+                                                        <xs:documentation>Abonnierte Buchungen.</xs:documentation>
+                                                </xs:annotation>
+                                        </xs:element>
+                                </xs:sequence>
+                        </xs:extension>
+                </xs:complexContent>
+        </xs:complexType>
+        <xs:complexType name="CompleteConsumptionRequestType">
+                <xs:annotation>
+                        <xs:documentation>Anfrage um den vollständigen aktuellen Status abonnierter Verbrauchsdaten zu erhalten.</xs:documentation>
+                </xs:annotation>
+                <xs:sequence>
+                        <xs:element name="MaxResults" type="xs:integer" minOccurs="0">
+                                <xs:annotation>
+                                        <xs:documentation>Definiert die maximale Anzahl Resultate, die in einer Nachricht zurückgegeben werden.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                </xs:sequence>
+        </xs:complexType>
+        <xs:complexType name="CompleteConsumptionResponseType">
+                <xs:annotation>
+                        <xs:documentation>Nachricht um den vollständigen aktuellen Status abonnierter Verbauchsdaten zu erhalten. Verschiedene solcher Antworten können zu einer einzigen Anfrage gehören.</xs:documentation>
+                </xs:annotation>
+                <xs:complexContent>
+                        <xs:extension base="AbstractBaseResponseType">
+                                <xs:sequence>
+                                        <xs:element name="MessageBlockID" type="xs:NMTOKEN">
+                                                <xs:annotation>
+                                                        <xs:documentation>ID des Blocks, zu dem diese Nachricht gehört. Alle Nachrichten eines Blockes haben dieselbe ID.</xs:documentation>
+                                                </xs:annotation>
+                                        </xs:element>
+                                        <xs:element name="Last" type="xs:boolean" default="false" minOccurs="0">
+                                                <xs:annotation>
+                                                        <xs:documentation>Falls gesetzt, ist diese Nachricht die Letzte im korrespondierenden Block. Andernfalls folgen mehr Nachrichten desselben Blocks.</xs:documentation>
+                                                </xs:annotation>
+                                        </xs:element>
+                                        <xs:element name="BookingChange" type="BookingChangeType" minOccurs="0" maxOccurs="unbounded"/>
+                                </xs:sequence>
+                        </xs:extension>
+                </xs:complexContent>
+        </xs:complexType>
+        <xs:complexType name="ConsumptionPushMessageType">
+                <xs:annotation>
+                        <xs:documentation>Push-Nachricht, die Verbrauchsdaten für abonnierte Buchungen enthält.</xs:documentation>
+                </xs:annotation>
+                <xs:sequence>
+                        <xs:element name="Conumption" type="ConsumptionType" maxOccurs="unbounded"/>
+                </xs:sequence>
+        </xs:complexType>
 	<xs:annotation>
 		<xs:documentation>**** Nachrichtencontainer ****</xs:documentation>
 	</xs:annotation>
