diff --git a/xml/IXSI.xsd b/xml/IXSI.xsd
index c14b44e..ce27573 100644
--- a/xml/IXSI.xsd
+++ b/xml/IXSI.xsd
@@ -383,6 +383,12 @@
 		</xs:annotation>
 		<xs:restriction base="xs:NMTOKEN"/>
 	</xs:simpleType>
+        <xs:simpleType name="ConsumptionIDType">
+                <xs:annotation>
+                        <xs:documentation>Enumeration for consumption classes.</xs:documentation>
+                </xs:annotation>
+                <xs:restriction base="xs:NMTOKEN"/>
+        </xs:simpleType>
 	<xs:simpleType name="AttributeIDType">
 		<xs:annotation>
 			<xs:documentation>Enumeration for attribute codes.</xs:documentation>
@@ -854,6 +860,43 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:complexType>
+        <xs:complexType name="ConsumptionType">
+                <xs:annotation>
+                        <xs:documentation>Type for single Consumption item.</xs:documentation>
+                </xs:annotation>
+                <xs:sequence>
+                        <xs:element name="Type" type="ConsumptionIDType">
+                                <xs:annotation>
+					<xs:documentation>Type of consumption.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                        <xs:element name="Name" type="xs:string">
+                                <xs:annotation>
+                                        <xs:documentation>Printable name of consumption item.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                        <xs:element name="Unit" type="xs:string" minOccurs="0">
+                                <xs:annotation>
+                                        <xs:documentation>Printable name of Unit.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                        <xs:element name="Value" type="xs:string">
+                                <xs:annotation>
+                                        <xs:documentation>Consumption value.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                        <xs:element name="Price" type="EuroPriceType" minOccurs="0">
+                                <xs:annotation>
+                                        <xs:documentation>Price in Euro-Cent.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                        <xs:element name="Description" type="xs:string" minOccurs="0">
+                                <xs:annotation>
+                                        <xs:documentation>Printable description.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                </xs:sequence>
+        </xs:complexType>
 	<xs:complexType name="OriginDestType">
 		<xs:annotation>
 			<xs:documentation>Type for an origin or a destination location.</xs:documentation>
@@ -1483,6 +1526,97 @@
 			</xs:extension>
 		</xs:complexContent>
 	</xs:complexType>
+        <xs:annotation>
+                <xs:documentation>**** Service (7) : Consumption data subscription ****</xs:documentation>
+        </xs:annotation>
+        <xs:complexType name="ConsumptionSubscriptionRequestType">
+                <xs:annotation>
+                        <xs:documentation>Request to create or cancel a consumption data subscription.</xs:documentation>
+                </xs:annotation>
+                <xs:sequence>
+                        <xs:element name="BookingID" type="BookingIDType" maxOccurs="unbounded">
+                                <xs:annotation>
+                                        <xs:documentation>Booking ID to subscribe for.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                        <xs:element name="Unsubscription" type="xs:boolean" default="false" minOccurs="0">
+                                <xs:annotation>
+                                        <xs:documentation>If true, cancel subscription.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                </xs:sequence>
+        </xs:complexType>
+        <xs:complexType name="ConsumptionSubscriptionResponseType">
+                <xs:annotation>
+                        <xs:documentation>Response to consumption data subscription.</xs:documentation>
+                </xs:annotation>
+                <xs:complexContent>
+                        <xs:extension base="AbstractBaseResponseType"/>
+                </xs:complexContent>
+        </xs:complexType>
+        <xs:complexType name="ConsumptionSubscriptionStatusRequestType">
+                <xs:annotation>
+                        <xs:documentation>Request for getting all subscribed consumption data items.</xs:documentation>
+                </xs:annotation>
+        </xs:complexType>
+        <xs:complexType name="ConsumptionSubscriptionStatusResponseType">
+                <xs:annotation>
+                        <xs:documentation>Response for getting all subscribed consumption data items.</xs:documentation>
+                </xs:annotation>
+                <xs:complexContent>
+                        <xs:extension base="AbstractBaseResponseType">
+                                <xs:sequence>
+                                        <xs:element name="BookingID" type="BookingIDType" minOccurs="0" maxOccurs="unbounded">
+                                                <xs:annotation>
+                                                        <xs:documentation>Subscribed bookings.</xs:documentation>
+                                                </xs:annotation>
+                                        </xs:element>
+                                </xs:sequence>
+                        </xs:extension>
+                </xs:complexContent>
+        </xs:complexType>
+        <xs:complexType name="CompleteConsumptionRequestType">
+                <xs:annotation>
+                        <xs:documentation>Request for getting the complete current status of subscribed consumption data items.</xs:documentation>
+                </xs:annotation>
+                <xs:sequence>
+                        <xs:element name="MaxResults" type="xs:integer" minOccurs="0">
+                                <xs:annotation>
+                                        <xs:documentation>Defines the maximum number of results to be returned in one message.</xs:documentation>
+                                </xs:annotation>
+                        </xs:element>
+                </xs:sequence>
+        </xs:complexType>
+        <xs:complexType name="CompleteConsumptionResponseType">
+                <xs:annotation>
+                        <xs:documentation>Message for getting the complete current status of subscribed consumption data. Several such responses may belong to one single request.</xs:documentation>
+                </xs:annotation>
+                <xs:complexContent>
+                        <xs:extension base="AbstractBaseResponseType">
+                                <xs:sequence>
+                                        <xs:element name="MessageBlockID" type="xs:NMTOKEN">
+                                                <xs:annotation>
+                                                        <xs:documentation>ID of the block, this message belongs to. All messages of one block have the same ID.</xs:documentation>
+                                                </xs:annotation>
+                                        </xs:element>
+                                        <xs:element name="Last" type="xs:boolean" default="false" minOccurs="0">
+                                                <xs:annotation>
+                                                        <xs:documentation>If set, this message is the last one in the corresponding block. Otherwise more messages of the same block will follow</xs:documentation>
+                                                </xs:annotation>
+                                        </xs:element>
+                                        <xs:element name="BookingChange" type="BookingChangeType" minOccurs="0" maxOccurs="unbounded"/>
+                                </xs:sequence>
+                        </xs:extension>
+                </xs:complexContent>
+        </xs:complexType>
+        <xs:complexType name="ConsumptionPushMessageType">
+                <xs:annotation>
+                        <xs:documentation>Push message containing alerts for subscribed bookings.</xs:documentation>
+                </xs:annotation>
+                <xs:sequence>
+                        <xs:element name="Conumption" type="ConsumptionType" maxOccurs="unbounded"/>
+                </xs:sequence>
+        </xs:complexType>
 	<xs:annotation>
 		<xs:documentation>**** Message container ****</xs:documentation>
 	</xs:annotation>
