--- IXSI_4.5.1.xsd	2014-10-28 11:05:32.036622090 +0100
+++ IXSI_4.6.xsd	2014-11-26 09:22:58.976671916 +0100
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Mit XMLSpy v2013 sp1 (x64) (http://www.altova.com) von Peter von Grumbkow (HaCon Ingenieurgesellschaft mbH) bearbeitet -->
 <!-- edited with XMLSpy v2010 rel. 3 sp1 (x64) (http://www.altova.com) by Peter von Grumbkow (HaCon Ingenieurgesellschaft mbH) -->
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified">
+<xs:schema xmlns="http://www.ixsi-schnittstelle.de/" xmlns:xs="http://www.w3.org/2001/XMLSchema" targetNamespace="http://www.ixsi-schnittstelle.de/" elementFormDefault="qualified" attributeFormDefault="unqualified">
 	<!--*******************************************************************-->
 	<!--Interface for X-Sharing Information (IXSI)-->
 	<!--*******************************************************************-->
-	<!--Spezifikation, Version 4 vom 2014-07-03-->
+	<!--Spezifikation, Version 4 vom 2014-11-25-->
 	<!--Urheber: Peter von Grumbkow, HaCon Ing.-Ges. mbH und Dirk Hillbrecht, cantamen GmbH-->
 	<!--Diese Schnittstellenspezifikation steht unter folgender Lizenz: CC BY-ND 3.0 DE (Creative Commons Namensnennung-Keine Bearbeitung 3.0 Deutschland), http://creativecommons.org/licenses/by-nd/3.0/de/-->
 	<!--*******************************************************************-->
-	<!--Specification, Version 4 of 2014-07-03-->
+	<!--Specification, Version 4 of 2014-11-25-->
 	<!--Authors: Peter von Grumbkow, HaCon Ing.-Ges. mbH and Dirk Hillbrecht, cantamen GmbH-->
 	<!--This interface specification is licensed according to: CC BY-ND 3.0 DE (Creative Commons Namensnennung-Keine Bearbeitung 3.0 Deutschland), http://creativecommons.org/licenses/by-nd/3.0/de/-->
 	<!--*******************************************************************-->
@@ -20,61 +20,61 @@
 		<xs:annotation>
 			<xs:documentation>Type for area IDs.</xs:documentation>
 		</xs:annotation>
-		<xs:restriction base="xs:NMTOKEN"/>
+		<xs:restriction base="xs:token"/>
 	</xs:simpleType>
 	<xs:simpleType name="PlaceIDType">
 		<xs:annotation>
 			<xs:documentation>Type for place IDs.</xs:documentation>
 		</xs:annotation>
-		<xs:restriction base="xs:NMTOKEN"/>
+		<xs:restriction base="xs:token"/>
 	</xs:simpleType>
 	<xs:simpleType name="PlaceGroupIDType">
 		<xs:annotation>
 			<xs:documentation>Type for place group IDs.</xs:documentation>
 		</xs:annotation>
-		<xs:restriction base="xs:NMTOKEN"/>
+		<xs:restriction base="xs:token"/>
 	</xs:simpleType>
 	<xs:simpleType name="BookeeIDType">
 		<xs:annotation>
 			<xs:documentation>Type for bookee IDs.</xs:documentation>
 		</xs:annotation>
-		<xs:restriction base="xs:NMTOKEN"/>
+		<xs:restriction base="xs:token"/>
 	</xs:simpleType>
 	<xs:simpleType name="ProviderIDType">
 		<xs:annotation>
 			<xs:documentation>Type for provider IDs.</xs:documentation>
 		</xs:annotation>
-		<xs:restriction base="xs:NMTOKEN"/>
+		<xs:restriction base="xs:token"/>
 	</xs:simpleType>
 	<xs:simpleType name="SessionIDType">
 		<xs:annotation>
 			<xs:documentation>Type for session IDs.</xs:documentation>
 		</xs:annotation>
-		<xs:restriction base="xs:NMTOKEN"/>
+		<xs:restriction base="xs:token"/>
 	</xs:simpleType>
 	<xs:simpleType name="BookingIDType">
 		<xs:annotation>
 			<xs:documentation>Type for booking IDs.</xs:documentation>
 		</xs:annotation>
-		<xs:restriction base="xs:NMTOKEN"/>
+		<xs:restriction base="xs:token"/>
 	</xs:simpleType>
 	<xs:simpleType name="UserIDType">
 		<xs:annotation>
 			<xs:documentation>Type for user IDs.</xs:documentation>
 		</xs:annotation>
-		<xs:restriction base="xs:NMTOKEN"/>
+		<xs:restriction base="xs:token"/>
 	</xs:simpleType>
 	<xs:simpleType name="SystemIDType">
 		<xs:annotation>
 			<xs:documentation>Type for system IDs.</xs:documentation>
 		</xs:annotation>
-		<xs:restriction base="xs:NMTOKEN"/>
+		<xs:restriction base="xs:token"/>
 	</xs:simpleType>
 	<xs:simpleType name="GlobalIDType">
 		<xs:annotation>
 			<xs:documentation>Type for globally unique IDs for identifiying clone entities.</xs:documentation>
 		</xs:annotation>
-		<xs:restriction base="xs:NMTOKEN"/>
+		<xs:restriction base="xs:token"/>
 	</xs:simpleType>
 	<xs:annotation>
 		<xs:documentation>**** General type definitions ****</xs:documentation>
@@ -203,12 +203,12 @@
 		<xs:sequence>
 			<xs:element name="Duration" type="xs:duration">
 				<xs:annotation>
-					<xs:documentation>Probable duration for getting a booking target. </xs:documentation>
+					<xs:documentation>Probable duration for finding a booking target. </xs:documentation>
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="Variance" type="xs:duration" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Variance of the probable duration for getting a booking target. The probable duration can vary by this value up or down.</xs:documentation>
+					<xs:documentation>Variance of the probable duration for finding a booking target. The probable duration can vary by this value up or down.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 		</xs:sequence>
@@ -226,6 +226,50 @@
 			<xs:group ref="DurationGroup"/>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:complexType name="AddressType">
+		<xs:annotation>
+			<xs:documentation>Type for storing an address.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="Country" type="xs:string" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Name of the country.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="PostalCode" type="xs:string" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Complete postal code of the address.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="City" type="xs:string">
+				<xs:annotation>
+					<xs:documentation>Name of the city.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="StreetHouseNr" type="xs:string" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Street and house number in one string.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="GeoPositionType">
+		<xs:annotation>
+			<xs:documentation>Type for a geopraphical position.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="Coord" type="CoordType">
+				<xs:annotation>
+					<xs:documentation>Geographical coordinates.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Address" type="AddressType" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Optional address of the geographical position.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+		</xs:sequence>
+	</xs:complexType>
 	<xs:complexType name="TimePeriodType">
 		<xs:annotation>
 			<xs:documentation>Type for a time period.</xs:documentation>
@@ -341,7 +385,7 @@
 		<xs:annotation>
 			<xs:documentation>Enumeration for error codes.</xs:documentation>
 		</xs:annotation>
-		<xs:restriction base="xs:NMTOKEN"/>
+		<xs:restriction base="xs:token"/>
 	</xs:simpleType>
 	<xs:complexType name="ErrorType">
 		<xs:annotation>
@@ -377,25 +421,25 @@
 		<xs:annotation>
 			<xs:documentation>Enumeration for car classes.</xs:documentation>
 		</xs:annotation>
-		<xs:restriction base="xs:NMTOKEN"/>
+		<xs:restriction base="xs:token"/>
 	</xs:simpleType>
 	<xs:simpleType name="EngineType">
 		<xs:annotation>
 			<xs:documentation>Enumeration for engines.</xs:documentation>
 		</xs:annotation>
-		<xs:restriction base="xs:NMTOKEN"/>
+		<xs:restriction base="xs:token"/>
 	</xs:simpleType>
 	<xs:simpleType name="AttributeClassType">
 		<xs:annotation>
 			<xs:documentation>Enumeration for attribute classes.</xs:documentation>
 		</xs:annotation>
-		<xs:restriction base="xs:NMTOKEN"/>
+		<xs:restriction base="xs:token"/>
 	</xs:simpleType>
 	<xs:simpleType name="AttributeIDType">
 		<xs:annotation>
 			<xs:documentation>Enumeration for attribute codes.</xs:documentation>
 		</xs:annotation>
-		<xs:restriction base="xs:NMTOKEN"/>
+		<xs:restriction base="xs:token"/>
 	</xs:simpleType>
 	<xs:complexType name="AttributeType">
 		<xs:annotation>
@@ -486,7 +530,7 @@
 			<xs:documentation>Type for link from a car-sharing place to a stop.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element name="StopID" type="xs:NMTOKEN"/>
+			<xs:element name="StopID" type="xs:token"/>
 			<xs:group ref="DurationGroup"/>
 			<xs:element name="Distance" type="xs:nonNegativeInteger" minOccurs="0">
 				<xs:annotation>
@@ -570,7 +614,7 @@
 					<xs:documentation>Globallly unique ID of this object. Used for identifying clones of this object, if they are delivered from different systems. Such ID should be formed using unique elements like DNS domain names.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="Coord" type="CoordType">
+			<xs:element name="GeoPosition" type="GeoPositionType">
 				<xs:annotation>
 					<xs:documentation>Geo position of the place.</xs:documentation>
 				</xs:annotation>
@@ -581,6 +625,11 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:group ref="PlaceOrAreaGroup"/>
+			<xs:element name="OnPremisesTime" type="xs:duration" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Additional time needed for finding the place on the premises. Can be used for large parking areas or parking houses. Defaults to 0. </xs:documentation>
+				</xs:annotation>
+			</xs:element>
 		</xs:sequence>
 	</xs:complexType>
 	<xs:complexType name="ProbabilityPlaceIDType">
@@ -694,6 +743,11 @@
 					<xs:documentation>Grid for start and end time of bookings in minutes. Should be a divisor of 60. Examples: 1, 5, 10, 15, 30, 60.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="OpeningTime" type="xs:duration" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Time needed for actually opening the booking target. If not given, the client system will assume some reasonable time.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
 			<xs:element name="Engine" type="EngineType" minOccurs="0"/>
 			<xs:element name="CO2Factor" type="xs:nonNegativeInteger" minOccurs="0">
 				<xs:annotation>
@@ -745,7 +799,7 @@
 					<xs:documentation>For currently available booking targets, which are not bound to a specific place, the current place should be returned.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="GeoPosition" type="CoordType" minOccurs="0">
+			<xs:element name="GeoPosition" type="GeoPositionType" minOccurs="0">
 				<xs:annotation>
 					<xs:documentation>For currently available booking targets, which are not place bounded, the current position should be returned.</xs:documentation>
 				</xs:annotation>
@@ -782,7 +836,7 @@
 					<xs:documentation>For currently available booking targets, which are not bound to a specific place, the current place should be returned.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="GeoPosition" type="CoordType" minOccurs="0">
+			<xs:element name="GeoPosition" type="GeoPositionType" minOccurs="0">
 				<xs:annotation>
 					<xs:documentation>For newly available vehicles, which are not place bounded, the position should be returned.</xs:documentation>
 				</xs:annotation>
@@ -809,7 +863,7 @@
 			<xs:element name="ID" type="ProviderPlaceIDType"/>
 			<xs:element name="Availability" type="xs:nonNegativeInteger">
 				<xs:annotation>
-					<xs:documentation>Currently available capacity for returning booking targets.</xs:documentation>
+					<xs:documentation>Currently available capacity for returning booking targets. This is the number of booking targets which could be returned now, without exceeding the overall capacity of the place.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 		</xs:sequence>
@@ -1238,7 +1292,7 @@
 		<xs:complexContent>
 			<xs:extension base="AbstractBaseResponseType">
 				<xs:sequence>
-					<xs:element name="MessageBlockID" type="xs:NMTOKEN">
+					<xs:element name="MessageBlockID" type="xs:token">
 						<xs:annotation>
 							<xs:documentation>ID of the block, this message belongs to. All messages of one block have the same ID.</xs:documentation>
 						</xs:annotation>
@@ -1329,7 +1383,7 @@
 		<xs:complexContent>
 			<xs:extension base="AbstractBaseResponseType">
 				<xs:sequence>
-					<xs:element name="MessageBlockID" type="xs:NMTOKEN">
+					<xs:element name="MessageBlockID" type="xs:token">
 						<xs:annotation>
 							<xs:documentation>ID of the block, this message belongs to. All messages of one block have the same ID.</xs:documentation>
 						</xs:annotation>
@@ -1493,7 +1547,7 @@
 		<xs:complexContent>
 			<xs:extension base="AbstractBaseResponseType">
 				<xs:sequence>
-					<xs:element name="MessageBlockID" type="xs:NMTOKEN">
+					<xs:element name="MessageBlockID" type="xs:token">
 						<xs:annotation>
 							<xs:documentation>ID of the block, this message belongs to. All messages of one block have the same ID.</xs:documentation>
 						</xs:annotation>
