--- IXSI_4.3.xsd	2014-07-18 11:09:45.803360401 +0200
+++ IXSI_4.4.xsd	2014-08-21 15:54:28.640184723 +0200
@@ -1830,6 +1830,11 @@
 					<xs:documentation>Transaction identifier, set by the requestor.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="SystemID" type="SystemIDType">
+				<xs:annotation>
+					<xs:documentation>ID of the requesting system.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
 			<xs:choice>
 				<xs:group ref="StaticDataRequestGroup"/>
 				<xs:group ref="UserTriggeredRequestGroup"/>
@@ -1867,6 +1872,11 @@
 					<xs:documentation>Transaction identifier, set by the requestor.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="SystemID" type="SystemIDType">
+				<xs:annotation>
+					<xs:documentation>ID of the requesting system.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
 			<xs:choice>
 				<xs:group ref="SubscriptionAdministrationRequestGroup"/>
 				<xs:group ref="SubscriptionRequestGroup"/>
